packages/0xcert-ethereum-gateway/src/lib/actions-order.ts

```diff
@@ -33,96 +30,114 @@ export function createOrderHash(gateway: Gateway, order: ActionsOrder) {
     hexToBytes([
       '0x',
       gateway.config.actionsOrderId.substr(2),
-      order.makerId.substr(2),
-      order.takerId.substr(2),
+      addressArrayParse(order.signers),
       temp.substr(2),
       leftPad(toInteger(order.seed), 64, '0', false),
       leftPad(toSeconds(order.expiration), 64, '0', false),
     ].join('')),
   );
 }
 
+export function addressArrayParse(addresses: string[]) {
```

Comment:
rename to `parseAddresses` or `parseOrderIds` since we use word "ID" or smth and add comment.